### PR TITLE
Read files when generating cluster pair and populate inline

### PR DESCRIPTION
**Pull request by disrani-px**

### CODE REVIEW

File: pkg/storkctl/clusterpair.go [hunk 1]
```diff
@@ -125,6 +149,36 @@ func newGenerateClusterPairCommand(cmdFactory Factory, ioStreams genericclioptio
 					}
 				}
 			}
+
+			if config.AuthInfos[currentAuthInfo].ClientCertificate != "" && len(config.AuthInfos[currentAuthInfo].ClientCertificateData) == 0 {
```
Comment (harsh-px) [inline pkg/storkctl/clusterpair.go]: If `config.AuthInfos[currentAuthInfo]` is nil, this will panic.

Comment (disrani-px) [reply pkg/storkctl/clusterpair.go]: The config loader already validates that the config is usable, ie all maps have valid keys, so don't need the check here.

Comment (harsh-px) [reply pkg/storkctl/clusterpair.go]: In that case, we can remove the `config.AuthInfos[currentAuthInfo] != nil` check from line 144.

Comment (disrani-px) [reply pkg/storkctl/clusterpair.go]: Actually looks like it is required if there are no auth infos at all in the config. Updated.

**Pull request merged**